swift/apple/FirezoneNetworkExtension/Adapter.swift

```diff
@@ -35,20 +31,13 @@ public enum AdapterError: Error {
 private enum AdapterState: CustomStringConvertible {
   case startingTunnel(session: WrappedSession, onStarted: Adapter.StartTunnelCompletionHandler?)
   case tunnelReady(session: WrappedSession)
-  case stoppingTunnel(session: WrappedSession, onStopped: Adapter.StopTunnelCompletionHandler?)
   case stoppedTunnel
-  case stoppingTunnelTemporarily(
-    session: WrappedSession, onStopped: Adapter.StopTunnelCompletionHandler?)
-  case stoppedTunnelTemporarily
 
   var description: String {
     switch self {
     case .startingTunnel: return "startingTunnel"
     case .tunnelReady: return "tunnelReady"
-    case .stoppingTunnel: return "stoppingTunnel"
     case .stoppedTunnel: return "stoppedTunnel"
-    case .stoppingTunnelTemporarily: return "stoppingTunnelTemporarily"
-    case .stoppedTunnelTemporarily: return "stoppedTunnelTemporarily"
```

Comment:
@roop As it turns out, we don't need to track state as closely if connlib is smarter about reconnect logic and we just use the path update callback for signaling to the user